### ISSUE TITLE
[12/edge] The PHP OPcache is not properly configured

### ISSUE THREAD

**Issue by SkyWheel** [nextcloud/nextcloud-snap#344]
Hi guys,

[12/Edge channel] Admin web console Basic settings shows notification:
```
The PHP OPcache is not properly configured. For better performance we recommend to use following settings in the php.ini:

opcache.enable=1
opcache.enable_cli=1
opcache.interned_strings_buffer=8
opcache.max_accelerated_files=10000
opcache.memory_consumption=128
opcache.save_comments=1
opcache.revalidate_freq=1
```

Is it misconfiguration? Or it should be like that?
Thanks.

**Comment by kyrofa** [comment]
Yeah this is a new waning provided by v12. I expect that it's valid, and once we actually move to v12 for real we'll fix it.

**Comment by kyrofa** [comment]
This is fixed in the v12 release, 12.0.4snap1.
